docs/organizations/overview.mdx

```diff
@@ -5,26 +5,107 @@ description: Learn about how to create and manage Clerk organizations and their
+### Roles and permissions
 
-<Callout type="info">
-  To explore Clerk's Organizations, check out this demo repo:
-  https://github.com/clerk/organizations-demo
-</Callout>
+Roles determine a user's level of access and permissions within an organization. [Learn more about how roles and permissions work and how to create your own with Clerk.](/docs/organizations/roles-permissions).
 
-## Roles and permissions
+### Default membership limit
 
-Roles determine a user's level of access and permissions within an organization. [Learn more about how roles and permissions work and how to create your own with Clerk.](/docs/organizations/roles-permissions).
+There is no limit to the number of organizations a user can be a member of.
+
+However, there is a limit to how many members total can be in a single organization. By default, the membership limit is set to 3 members. To change this limit, scroll to the **Default membership limit** section and update the membership limit.
+
+If you are on the Free plan, you can update the membership limit to a maximum of 5 members.
+
+If you have the Pro plan, you can set the membership limit to unlimited.
+
+You can also change this limit on a per-organization basis:
+
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=organizations).
+2. In the navigation sidebar, select **Organizations**.
+3. Select the organization you want to update.
+4. In the **Membership limit** section, update the membership limit.
+
+### Default ability to delete
+
+By default, organizations are deletable. Any member with the "Delete organization" permission can delete an organization. To prevent organizations from being deleted, you can disable the ability to delete organizations by scrolling to the **Default ability to delete** section and unchecking the option.
+
+### Verified domains
+
+Verified domains can be used to streamline enrollment into an organization. For example, if the domain `@clerk.com` is verified, any user with an email address ending in `@clerk.com` can be automatically invited or be suggested to join an organization with that domain. This feature is useful for organizations that want to restrict membership to users with specific email domains. See the [verified domains](/docs/organizations/verified-domains) documentation for more information.
+
+## Active organization
+
+There are two types of accounts that a user can have in a Clerk application: a personal account and an organization account. A personal account is the default account type, and it is created when a user signs up for your application. An organization account is created when a user creates or joins an organization within your application.
```

Comment:
Technically there are not two types of accounts. It's a single account that operates under different contexts, personal and organization related. We should make sure that customers don't get confused and think those are separated accounts